api/tableau_api/wdc.py

```diff
@@ -0,0 +1,43 @@
+from flask import render_template
+from flask.views import MethodView
+from flask_cors import cross_origin
+
+from api.tableau_api.constants import SERIALIZABLE_FIELD_NAMES, FIELD_TYPE_MAP
+from database.tableau_api_models import SummaryStatisticDaily
+
+class WebDataConnector(MethodView):
+
+    path = '/api/v0/studies/<string:study_id>/summary-statistics/daily/wdc'
```

Comment:
I feel like `/wdc/v0/...` would be slightly better so that the url paths remain consistent between the two. Thoughts?